src/lattice/reciprocal/lattice_reciprocal.js

```diff
@@ -69,4 +87,28 @@ export class ReciprocalLattice extends Lattice {
         });
         return kpointPath;
     }
+
+    /**
+     * Calculate grid dimension based on reciprocal lattice vectors.
+     * @param nPoints - Total number of points
+     * @param index - Index of reciprocal vector
+     * @return {number} - Grid dimension in direction of reciprocal vector
+     * @todo This could be moved to a separate KGrid class.
+     */
+    calculateDimension(nPoints, index) {
+        const norms = this.reciprocalVectorNorms;
+        const [j, k] = [0, 1, 2].filter((i) => i !== index); // get indices of other two dimensions
+        const N = Math.cbrt((nPoints * norms[index] ** 2) / (norms[j] * norms[k]));
```

Comment:
Let's just use `math` maybe as we importing it everywhere? It wraps `Math` also, I think

Reply:
`math` wraps [math.js](https://mathjs.org/docs/reference/functions.html) v3.20.2, but not `Math`. Ok, I'll change it to use `math` only.